src/test/java/core/basesyntax/MainAppTest.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainAppTest {
+    public static void main(String[] args) {
+        final int MACHINE_COUNT = 3;
```

Comment:
```suggestion
    private static final int MACHINE_COUNT = 3;
    
    public static void main(String[] args) {  
```
Make this variable constant